components/nft-sale-widget/layout/nft-sale/mint-section.tsx

```diff
@@ -223,7 +133,8 @@ const MintSection = ({
             isPrivateSale={privateSaleStatus}
             isOriginal={isOriginal}
             loading={loading}
-            refetchBalances={refetchAllBalances}
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            refetchBalances={() => {}}
```

Comment:
just wondering, shouldn't we maybe remove this prop instead of passing an empty function?